internal/services/user_management/user_management.go

```diff
@@ -0,0 +1,108 @@
+package user_management
+
+import (
+	"Tradeasy/config"
+	"Tradeasy/internal/model"
+	"Tradeasy/internal/provider/redis"
+	"errors"
+	"strings"
+	"time"
+)
+
+func SignUp(SUpReq SignUpRequest) (SUpRes SignUpResponse, err error) {
+	email := SUpReq.EmailId
+	SUpRes.UserId = strings.Split(email, "@")[0]
+	SUpRes.Password = SUpReq.Password
+	SUpRes.Message = "User registered"
```

Comment:
Create response at the end of method if everything works fine.

Reply:
ok